### PR TITLE
Template update

### LINKED ISSUE #4: Update pipeline template
### Description of feature

Update to the latest nf-core pipeline template

### LINKED ISSUE #7: Add CI tests on the farm
### Description of feature

And a "cleanup" profile

### PR DESCRIPTION
**Pull request by muffato**
Closes #4
I've updated the pipeline template from 2.5 to 2.8.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/ensemblrepeatdownload/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by muffato** [comment]
Added a few commits to address issues actually found in https://github.com/sanger-tol/ensemblgenedownload/pull/6 and a commit to enable the `cleanup` profile for our Tower tests and close #7

**Review by priyanka-surana — APPROVED** [review]
LGTM 👍🏼 

I would recommend add the Rapid release website support only warning to the README as well.

Looking at this and gene download workflow, there seems to be a strong case for combining them. The parameters here are a subset of gene download. Most subworkflows are overlapping. And if there is still a need to keep the features apart, this can be easily done with a parameter like `--download_type repeat/gene`. The greatest advantage would be not having to maintain 2 very similar pipelines. It can be `ensembldonwload` similar to `insdcdownload`.

**Comment by priyanka-surana** [comment]
Unrelated query: What is this "Run pipeline with test data" CI that keeps running but does not complete? I saw this on `treeval` pipeline too.

**Comment by gq1** [comment]
> Unrelated query: What is this "Run pipeline with test data" CI that keeps running but does not complete? I saw this on `treeval` pipeline too.



> Unrelated query: What is this "Run pipeline with test data" CI that keeps running but does not complete? I saw this on `treeval` pipeline too.

This is one of the CI requirements we set through the pipeline website health page. Obviously we run it twice with the different version of nextflow, Github doesn't match them.  We can add the version to the requirement as well, probably just the latest version?

File: README.md [hunk 1]
```diff
@@ -19,14 +19,6 @@
 
 **sanger-tol/ensemblrepeatdownload** is a pipeline that downloads repeat annotations from Ensembl into a Tree of Life directory structure.
```
[the remark this answers is not in the document]
Comment (muffato) [reply README.md:20]: Yes. The relevant ToL directory structure for this pipeline is that the output files go into `repeats/ensembl/

**Comment by muffato** [comment]
I think your comments abut merging the pipelines make sense. I already have the changes regarding `--outdir` lined up, but I look into that after.

**Pull request merged**